Wireframe/index.html

```diff
@@ -4,10 +4,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
+      <header>
+        <h1>GIT</h1>
+        <P>(Information Tracker)</P>
+      </header>
+      <main>
+        <article>
+          <img src="//res.cloudinary.com/practicaldev/image/fetch/s--ZqwTtpA6--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/6vq45tzju4l6m3aijeoq.png" alt="A picture with three computers and different users explaining about git">
```

Comment:
good alt tag. 

Linking to a website on another site like this is called image hotlinking. It is considered a bad practice, because you are using another website's bandwidth and because the owner of the second website could delete or even overwrite the image (perhaps with an...unfortunate image).

Here's some more info https://www.hostinger.co.uk/tutorials/hotlinking